color/_tegel_brand_usage.scss

```diff
@@ -0,0 +1,292 @@
+// Tegel Brand Usage Tokens
+
+// Light mode tokens (applied to :root and .scania-light)
+:root,
+.scania-light {
```

Comment:
Rename this to tds-mode-light tds-mode-dark

Reply:
I think tds-mode-light / tds-mode-dark is better for consistency